common/changes/@microsoft.azure/openapi-validator-rulesets/tejaswis-tagsAreTopLevelPropertiesOnly_2024-06-13-22-19.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
```

Comment:
Note that you no longer need to include these changelog json files. See here:

https://github.com/Azure/azure-openapi-validator/issues/683

Reply:
Thanks konrad removed the file